packages/contracts/README.md

```diff
@@ -1,51 +1,61 @@
 # JWT Contracts
 
+This directory contains verifier contracts for handling JSON Web Tokens(JWTs) proofs. 
 
-Build
+## Contracts Overview
 
-```
-$ yarn build
-```
+### JWTRegistry
 
-Unit tests
+- **Purpose**: The `JWTRegistry` contract is used to store and manage JWTs. It keeps track of all issued tokens and their statuses.
+- **Usage**: In another project, you can use this contract to verify if a JWT is valid and to check its status (active, revoked, etc.).
 
-```
-$ yarn test:unit
-```
+### JWTVerifier
 
-Integration tests
+- **Purpose**: The `JWTVerifier` contract is responsible for verifying the authenticity of JWTs. It checks the proof and ensures the token is valid.
+- **Usage**: Use this contract in your project to verify JWTs proof before granting access to protected resources.
 
-```
-$ yarn test:integration
-```
+## How to Use in Another Project
+
+1. **Installation**: First, install the package using npm. Run the following command in your project directory:
+
+   ```
+   npm install @zk-jwt/zk-jwt-contracts
+   ```
 
-To deploy, clone .env.sample and fill in your API and private keys, then run:
+2. **Integration**: 
+   - **Solidity**: Import the contracts in your Solidity files using the following syntax:
+
+     ```solidity
+     import {JwtVerifier} from "@zk-jwt/zk-jwt-contracts/utils/JwtVerifier.sol";
+     import {JwtGroth16Verifier} from "@zk-jwt/zk-jwt-contracts/utils/JwtGroth16Verifier.sol";
+     import {JwtRegistry} from "@zk-jwt/zk-jwt-contracts/utils/JwtRegistry.sol";
+     ```
+
+## Example Deployment
+
+To deploy, run the following commands:
 
 ```
 foundryup
 source .env
 forge script script/Deploy_jwtRegistry.s.sol:DeployScript --rpc-url $RPC_URL --broadcast --verify -vvvv --sender $ETH_FROM
 ```
 
-Here is the sample verified deployed addresses on Sepolia:
-```
-JWTRegistry: 0x60Dd906E3D1d827d23Bd393aa3224fb38cac1A11
-JWTVerifier implementation: 0x020FD65080C114AB55Cdeb018db197f3A8751B7F
-JWTVerifier proxy: 0xD3863Ad6AD48e3dEc3736d335967b4117f64ce49
-```
+## Sample Deployed Addresses
 
-## Errors
+Here are some example addresses where the contracts have been deployed on Base Sepolia test network:
 
-```
-    ├─ [1040390199] Create2Deployer::create2()
-    │   ├─ [0] → new MetaMultiSigWallet@0x70EEAD58493d2cDcc62004674c95ee6C529723E1
-    │   │   └─ ← [CreateCollision] EvmError: CreateCollision
-    │   └─ ← [Revert] EvmError: Revert
-    └─ ← [Revert] EvmError: Revert
+- **JWTRegistry**: `0x60Dd906E3D1d827d23Bd393aa3224fb38cac1A11`
+- **JWTVerifier Implementation**: `0x020FD65080C114AB55Cdeb018db197f3A8751B7F`
+- **JWTVerifier Proxy**: `0xD3863Ad6AD48e3dEc3736d335967b4117f64ce49`
```

Comment:
@wshino Can you also add example code for people to integrate into their own contracts.

Reply:
I added example how to deploy and use it.